screenshots.sh

```diff
@@ -22,15 +22,4 @@ for PRODUCT in Focus Klar; do
           --languages "$LANGUAGES" \
           --output_directory "screenshots/$TS/$PRODUCT/$DEVICEDIR" \
           --clear_previous_screenshots > "screenshots/$TS/$PRODUCT/$DEVICEDIR/snapshot.log" 2>&1
-  done
 done
-
-echo "You can now move these to pmo:"
```

Comment:
Once this PR lands, we can focus on where/how to post the results. We have a few ideas in mind already